ex03.ex

```diff
@@ -101,8 +109,9 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal([], []), do: true
```

Comment:
No deduction, but the same thing applies as with 3.1. 

~~~ elixir
list_equal(:cat, 99)   #-> false
~~~

Ideally our functions should only be defined for their domain.